src/instance/mgn.go

```diff
@@ -53,13 +53,18 @@ func StartInstance(name string, configuration int64) error {
 	s := settings.GetSettings()
 
 	// write config
-	if err := writeConfig(config); err != nil {
+	iniServerCfg, iniEntryList, err := writeConfig(config)
+
+	if err != nil {
 		return util.OpError{3, "Error writing configuration"}
 	}
 
+	// force server_cfg and entry_list ini paths
 	// start
-	cmd := exec.Command(filepath.Join(s.Folder, s.Executable), strings.Split(s.Args, " ")...)
-	now := strings.Replace(time.Now().String(), " ", "_", -1)
+	// FIXME: s.Args has been  discarted. No real use so far?
```

Comment:
s.Args is used to pass start parameters from the UI (like `linux` shown in the demo video: https://www.youtube.com/watch?v=z6osf2BMKqE). Please add it as the last parameter to exec.Command(... (the split is required!).